src/apps/utep_sdg_11_3_1/styles/index.scss

```diff
@@ -0,0 +1,189 @@
+@import '../../../styles/variables';
+@import './variables';
+
+.ptr-light {
+
+	--accent0: #{$lightAccent0};
+	--accent0rgb: #{$lightAccent0rgb};
+	--accent05: #{$lightAccent05};
+	--accent10: #{$lightAccent10};
+	--accent15: #{$lightAccent15};
+	--accent20: #{$lightAccent20};
+	--accent25: #{$lightAccent25};
+	--accent30: #{$lightAccent30};
+	--accent35: #{$lightAccent35};
+	--accent40: #{$lightAccent40};
+	--accent45: #{$lightAccent45};
+	--accent50: #{$lightAccent50};
+	--accent50rgb: #{$lightAccent50rgb};
+	--accent55: #{$lightAccent55};
+	--accent60: #{$lightAccent60};
+	--accent65: #{$lightAccent65};
+	--accent70: #{$lightAccent70};
+	--accent75: #{$lightAccent75};
+	--accent80: #{$lightAccent80};
+	--accent85: #{$lightAccent85};
+	--accent90: #{$lightAccent90};
+	--accent95: #{$lightAccent95};
+	--accent100: #{$lightAccent100};
+	--accent100rgb: #{$lightAccent100rgb};
+}
+
+.utep_sdg_11_3_1-app {
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+}
+
+$headerHeight: m(2);
+
+.utep_sdg_11_3_1-header {
+	flex: 0 0 $headerHeight;
+	max-height: $headerHeight;
+	display: flex;
+	z-index: 20;
+}
+
+.utep_sdg_11_3_1-content {
+	flex: 1;
+	max-height: calc(100% - #{$headerHeight});
+	position: relative;
+}
+
+
+
+.utep_sdg_11_3_1-header-home {
+	flex: 0 0 7rem;
+	background-color: #fff;
+	position: relative;
+}
+.utep_sdg_11_3_1-header-home > a {
+	display: block;
+	width: 6rem;
+	height: 2.125rem;
+	padding: 0 .5rem;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%,-50%);
+	cursor: pointer;
+}
+.utep_sdg_11_3_1-header-home::after {
+	content: " ";
+	position: absolute;
+	top: 50%;
+	right: 0;
+	height: 60%;
+	width: 1px;
+	transform: translateY(-50%);
+	background-color: var(--base20);
+}
+.utep_sdg_11_3_1-header-region-select {
+	flex: 0 0 16rem;
+	background-color: #fff;
+}
+.utep_sdg_11_3_1-header-indicator-select {
```

Comment:
potřebujeme to ve stylech, když k tomu nemáme komponenty?

Reply:
promažu